README.md

```diff
@@ -26,6 +26,8 @@ Once you have this up and running. You should see the following output:
 ... - info: FHIR Server successfully started.
 ```
 
+Unzip Synthea data "\fhir\fixtures\data\data-shr.zip" into \fhir\fixtures\data\shr"
```

Comment:
We're including the data set as a zip file in the repo, rather than just including all the files?  I guess that makes sense to save space unless you need it, but it seems weird.  Are we at least including the unzipped directory in the `.gitignore`?

Reply:
Also, you might want to use forward slashes `/` instead of backslashes `\` which are specifically a Windows thing...

Reply:
Will use forward slashes. 
1000 unzipped files are about 260 Mbs, so Rob and I thought it might be better to zip them; however, we'd still need to discuss further since our test scripts require json files to populate the database properly.